tests/test_calibrateImage.py

```diff
@@ -0,0 +1,396 @@
+# This file is part of pipe_tasks.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+import tempfile
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+import numpy as np
+
+import lsst.afw.image as afwImage
+import lsst.afw.table as afwTable
+import lsst.daf.base
+import lsst.daf.butler.tests as butlerTests
+import lsst.geom
+import lsst.meas.algorithms
+from lsst.meas.algorithms import testUtils
+import lsst.meas.extensions.psfex
+import lsst.meas.base.tests
+import lsst.pipe.base.testUtils
+from lsst.pipe.tasks.calibrateImage import CalibrateImageTask
+import lsst.utils.tests
+
+
+class CalibrateImageTaskTests(lsst.utils.tests.TestCase):
+
+    def setUp(self):
+        # Different x/y dimensions so they're easy to distinguish in a plot.
+        bbox = lsst.geom.Box2I(lsst.geom.Point2I(0, 0), lsst.geom.Point2I(200, 180))
```

Comment:
While we don't need this in any production context I am aware of, it'd also be good to set the bbox min to something other than `(0, 0)` to catch xy0 usage errors.